backend/spring-routie/src/main/java/routie/routie/controller/RoutieController.java

```diff
@@ -34,4 +38,14 @@ public ResponseEntity<RoutieUpdateResponse> updateRoutie(
     ) {
         return ResponseEntity.ok(routieService.modifyRoutie(id, routieUpdateRequest));
     }
+
+    @GetMapping("/routies/{routieId}/validity")
+    public ResponseEntity<RoutieTimeValidationResponse> validateRoutieTime(
+            @PathVariable final Long routieId,
+            @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) final LocalDateTime startDateTime,
+            @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) final LocalDateTime endDateTime
+    ) {
+        RoutieTimeValidationResponse result = routieService.validateRoutie(routieId, startDateTime, endDateTime);
```

Comment:
특별한 의미가 없는 변수라면, 클래스명을 그대로 가져가는 것이 컨벤션이라 바꾸면 좋겠습니다.
```suggestion
        RoutieTimeValidationResponse routieTimeValidationResponse = routieService.validateRoutie(routieId, startDateTime, endDateTime);
```